purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -54,6 +58,15 @@ data class PaywallColor(
             null
         },
     )
+
+    constructor(@ColorInt colorInt: Int) : this(
+        stringRepresentation = String.format(Locale.US, COLOR_HEX_FORMAT, COLOR_WHITE_HEX and colorInt),
+        underlyingColor = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
This makes sense 👍🏻  I didn't do this in iOS cause it would be harder to extract the color (not to mention that only works with some color spaces.